### PR TITLE
[typo] Fixing Excel page title casing

### PR DESCRIPTION
**Pull request by AlexJerabek**
Making the [Chart](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-charts?view=office-js) and [Shape](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-shapes?view=office-js) article titles match the casing of the [Table](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-tables?view=office-js) and [Worksheet](https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-worksheets?view=office-js) articles.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9bb5573](https://github.com/OfficeDev/office-js-docs-pr/pull/1180/commits/9bb55730bd16622cc77b49a208cadfaa9661ca34)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172036243468-1180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=VfG5IoyPxuY9Ou9bedJM5H9iSnNkhv7gOjKDgSY1Kds%3D&st=2019-07-17T20%3A32%3A21Z&se=2019-08-17T20%3A37%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-shapes.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 description: ''
 ms.date: 04/30/2019
 localization_priority: Normal
 ---
 
-# Work with Shapes using the Excel JavaScript API
+# Work with shapes using the Excel JavaScript API
 
-Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape]/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with Charts using the Excel JavaScript API]](excel-add-ins-charts.md)).
+Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape]/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with charts using the Excel JavaScript API]](excel-add-ins-charts.md)).
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-shapes.md]: A number of these markdown links are not formatted correctly.

File: docs/excel/excel-add-ins-charts.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Work with Charts using the Excel JavaScript API
+title: Work with charts using the Excel JavaScript API
 description: ''
 ms.date: 03/19/2019
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-charts.md]: could also update the ms.date these pages

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0067137](https://github.com/OfficeDev/office-js-docs-pr/pull/1180/commits/00671375333b27bb53078bba5f5f1f81b52dd372)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172042365842-1180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=GhrNpRraM7OXuoeW%2BaEwy1lFy0lXALfgVsm8OVh4zew%3D&st=2019-07-17T20%3A38%3A35Z&se=2019-08-17T20%3A43%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. Not sure what was up with those links, but they're fixed now.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5c1f9a4](https://github.com/OfficeDev/office-js-docs-pr/pull/1180/commits/5c1f9a41d0376cc133a16708ebff43372223e43b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172044129774-1180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5FEjcJsgFJ1L5YQK8uBo7Eq03mu48MTMxJoE0i7mrQY%3D&st=2019-07-17T20%3A40%3A14Z&se=2019-08-17T20%3A45%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-shapes.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-ms.date: 04/30/2019
+ms.date: 07/19/2019
 localization_priority: Normal
 ---
 
-# Work with Shapes using the Excel JavaScript API
+# Work with shapes using the Excel JavaScript API
 
-Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape]/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with Charts using the Excel JavaScript API]](excel-add-ins-charts.md)).
+Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape](/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with charts using the Excel JavaScript API](excel-add-ins-charts.md)).
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-shapes.md]:
```suggestion
Excel defines shapes as any object that sits on the drawing layer of Excel. That means anything outside of a cell is a shape. This article describes how to use geometric shapes, lines, and images in conjunction with the [Shape](/javascript/api/excel/excel.shape) and [ShapeCollection](/javascript/api/excel/excel.shapecollection) APIs. [Charts](/javascript/api/excel/excel.chart) are covered in their own article, [Work with charts using the Excel JavaScript API](excel-add-ins-charts.md).
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f793275](https://github.com/OfficeDev/office-js-docs-pr/pull/1180/commits/f793275497415a6632d3da9f21e7775e4094a19a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-charts.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-charts.md) | :white_check_mark:Succeeded | |
[docs/excel/excel-add-ins-shapes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ExcelTitles/docs/excel/excel-add-ins-shapes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C17%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201907172052052585-1180%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jX7arUe3uO%2FgGk7Vrmodzj2P%2B7iSLS7Ruy7mC7Gpf58%3D&st=2019-07-17T20%3A48%3A16Z&se=2019-08-17T20%3A53%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-shapes.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
-title: Work with Shapes using the Excel JavaScript API
+title: Work with shapes using the Excel JavaScript API
 description: ''
-ms.date: 04/30/2019
+ms.date: 07/19/2019
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-shapes.md]: nit: should be 07/17/2019

Comment (AlexJerabek) [reply docs/excel/excel-add-ins-shapes.md]: Well... it probably won't get published until then :-)

**Pull request merged**